src/modules/organization/organization.dto.ts

```diff
@@ -70,6 +70,10 @@ export class OrganizationDTO implements BaseEnsEntity {
   @ApiProperty()
   namespace: string;
 
+  @IsString()
+  @ApiProperty()
+  namehash: string;
+
```

Comment:
I wonder if it makes sense to have the namehash be a `get` only property that we set ourselves by hashing the namespace? I'm just trying to achieve some validation that the namehash is actually correct for the namespace in question 🤔

Reply:
This makes sense But i think we should make that as an improvement on this for later. from experience it's not straight forward implementing setters and getters with typeorm entities.